source/transfer_list.rst

```diff
@@ -951,6 +951,48 @@ software running in Secure, Non-Secure, or Realm modes.
      - hdr_size
      - Holds a single `entry_point_info` structure.
 
+**FF-A SP binary (XFERLIST_FFA_SP_BINARY)**
+
+This entry holds a reference to an FF-A secure partition's binary.
```

Comment:
When secure partition is first used, you should capitalize and add (SP), then use SP from then on. Also SP needs spelling out in the commit message.

Reply:
Done